pkg/util/config.go

```diff
@@ -34,5 +34,6 @@ func DefaultResourceThresholdStrategy() *slov1alpha1.ResourceThresholdStrategy {
 		Enable:                      pointer.BoolPtr(false),
 		CPUSuppressThresholdPercent: pointer.Int64Ptr(65),
 		CPUSuppressPolicy:           slov1alpha1.CPUSetPolicy,
+		MemoryEvictThresholdPercent: pointer.Int64Ptr(70),
```

Comment:
add a default value here, but this field should be overridden in the nodeslo controller according to the configmap configuration.

Reply:
it will be overridden by `getResourceThresholdSpec`:
https://github.com/koordinator-sh/koordinator/blob/cb24c0804ec7ab53becef318ddf07513cfc974e4/pkg/slo-controller/nodeslo/resource_strategy.go#L33-L52

also it's verified by unit test here:
https://github.com/koordinator-sh/koordinator/blob/48f0ccc88cf917f55bec30f8f8110dadbaf275ba/pkg/slo-controller/nodeslo/resource_strategy_test.go#L36-L42
https://github.com/koordinator-sh/koordinator/blob/48f0ccc88cf917f55bec30f8f8110dadbaf275ba/pkg/slo-controller/nodeslo/resource_strategy_test.go#L143-L148